tests/test_km_distance_level.py

```diff
@@ -0,0 +1,243 @@
+import pandas as pd
+import pytest
+
+import splink.duckdb.duckdb_comparison_level_library as clld
+import splink.duckdb.duckdb_comparison_library as cld
+import splink.spark.spark_comparison_level_library as clls
+import splink.spark.spark_comparison_library as cls
+from splink.duckdb.duckdb_linker import DuckDBLinker
+from splink.spark.spark_linker import SparkLinker
+
+
+@pytest.mark.parametrize(
+    ("cl"),
+    [
+        pytest.param(cld, id="DuckDB Distance in KM simple run tests"),
+        pytest.param(cls, id="Spark Distance in KM simple run tests"),
+    ],
+)
+def test_simple_run(cl):
+
+    print(
+        cl.distance_in_km_at_thresholds(
+            lat_col="lat", long_col="long", km_thresholds=[1, 5, 10]
+        ).as_dict()
+    )
+
```

Comment:
```suggestion
    print(
        cl.distance_in_km_at_thresholds(
            lat_col="latlong[0]", long_col="latlong[1]", km_thresholds=[1, 5, 10]
        ).as_dict()
    )
    print(
        cl.distance_in_km_at_thresholds(
            lat_col="ll['lat']", long_col="ll['long']", km_thresholds=[1, 5, 10]
        ).as_dict()
    )

@pytest.mark.parametrize(
    ("cll"),
    [
        pytest.param(clld, id="DuckDB Distance in KM cll simple run tests"),
        pytest.param(clls, id="Spark Distance in KM cll simple run tests"),
    ],
)
def test_simple_run_cll(cll):
    print(
        cll.distance_in_km_level(
            lat_col="lat", long_col="long", km_threshold=1
        ).as_dict()
    )
    print(
        cll.distance_in_km_level(
            lat_col="latlong[0]", long_col="latlong[1]", km_threshold=0.1
        ).as_dict()
    )
    print(
        cll.distance_in_km_level(
            lat_col="ll['lat']", long_col="ll['long']", km_threshold=10
        ).as_dict()
    )
```
Checks for using array/object columns - as things stand the `cll` version passes, but the new comparison code does not